internal/dag/ingress_processor.go

```diff
@@ -58,7 +58,7 @@ func (p *IngressProcessor) Run(dag *DAG, source *KubernetesCache) {
 // computeSecureVirtualhosts populates tls parameters of
 // secure virtual hosts.
 func (p *IngressProcessor) computeSecureVirtualhosts() {
-	for _, ing := range p.source.ingresses {
+	for _, ing := range p.source.ingressesv1beta1 {
```

Comment:
I think we're going to need to check both `kc.ingresses` and `kc.ingressesv1beta1`, since either one of them could be populated based on whether `v1` exists in the cluster or not.